src/test/ui/ast-json/issue-78398-foreign-ice.rs

```diff
@@ -0,0 +1,18 @@
+// Regression test for issue #78398
+// Tests that we don't ICE when trying to print the AST json
+// when we have capturd tokens for a foreign item
+
+// check-pass
+// compile-flags: -Zast-json
```

Comment:
is there a way to dump this as pretty json? Any future changes causing a diff in that one line will be hard to review

Reply:
Not that I can tell - the output has the same format as the other tests in this directory.